pkg/controller/kubebuilderx/controller.go

```diff
@@ -75,7 +75,7 @@ func (c *controller) Prepare(reader TreeLoader) Controller {
 }
 
 func (c *controller) Do(reconcilers ...Reconciler) Controller {
-	if c.err != nil {
+	if c.err != nil || c.res.Next == cmmt || c.res.Next == rtry {
```

Comment:
```
c.res.Next != cntn 
```
looks easier to understand

Reply:
both `cntn` and `""`(empty next action) are treated as a `Continue` semantic.

Reply:
shall we allow empty next action?

Reply:
force next action to be one of `Continue`, `Commit` or `Retry`: [0179929](https://github.com/apecloud/kubeblocks/pull/7839/commits/017992926a7de0e41c7ccbbac63513cdfa15c0b5)